README.md

```diff
@@ -1008,6 +1008,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Siddharth Singh Chouhan](https://github.com/Sid672)
   - [Siddharth Singh Tanwar](https://github.com/SiddharthSinghTanwar)
   - [Sideon Kaves](https://github.com/Radiant690)
+  - [Sienna Lind] (https://https://github.com/siennalind)
```

Comment:
There's an extra space here that must be removed.
```suggestion
  - [Sienna Lind](https://https://github.com/siennalind)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Sorry, didn't notice the space in other names! Removed the space and the extra 'https' in the link